coroutine/ppc/Context.S

```diff
@@ -7,66 +7,66 @@
 .globl PREFIXED_SYMBOL(SYMBOL_PREFIX,coroutine_transfer)
 PREFIXED_SYMBOL(SYMBOL_PREFIX,coroutine_transfer):
 	# Make space on the stack for caller registers
-	addi r1,r1,-80
+	addi %r1,%r1,-80
```

Comment:
LLVM doesn't like rX syntax. It either allows X syntax, like `addi 1,1,-80`, but it may make the code less readable and I think old binutils from Mac OS X 10.4 or 10.5 may not like it, or %rX.

Reply:
So this change is incompatible with Mac OS X <= 10.5?

Reply:
@ioquatix For OS <= 10.6. `ppc32` is supported in 10.6 and below, `ppc64` only in 10.5.

Darwin does not use % for registers (see any example on Apple Opensource).

@pkubaj Please do not do this, it gonna break the build on Mac. If BSD or Linux uses something else, different implementation should be made, separately, or via setting tags in the file, so that each OS has correct syntax.